From fe5eeb25e2575712cb8e6569fbc2b75767e41d72 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <mbridon@uperto24.(none)>
Date: Thu, 6 Mar 2008 11:29:21 +0100
Subject: [PATCH 1/1] Fix for issue321: labels uncorrectly set.

In document viewer, labels for browsing the pages were set to other labels.
---
 .../conference/whiteboard/toolbar/documentMenu.lzx |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/client/conference/whiteboard/toolbar/documentMenu.lzx b/client/conference/whiteboard/toolbar/documentMenu.lzx
index ba8b781..76738fc 100644
--- a/client/conference/whiteboard/toolbar/documentMenu.lzx
+++ b/client/conference/whiteboard/toolbar/documentMenu.lzx
@@ -9,13 +9,13 @@
     <drawChooseSeperator y="4" />
 	
     <drawDocumentItem y="4" name="b3" modi="doc_first" resourceN="doc_first" 
-        labelid="254" />		
+        labelid="255" />		
     <drawDocumentItem y="4" name="b4" modi="doc_pre" resourceN="doc_pre" 
-        labelid="255" />			
+        labelid="256" />			
     <drawDocumentItem y="4" name="b5" modi="doc_next" resourceN="doc_next" 
-        labelid="256" />		
-    <drawDocumentItem y="4" name="b6" modi="doc_last" resourceN="doc_last" 
         labelid="257" />		
+    <drawDocumentItem y="4" name="b6" modi="doc_last" resourceN="doc_last" 
+        labelid="258" />		
     <view width="4" />
     <customEdittext name="_currentSlideNo" y="4" width="40" />	 
     <labelText y="6" labelid="254" />	  
-- 
1.5.4.1

